components/integrations/bots/bots.tsx

```diff
@@ -1,98 +1,108 @@
 
         /**
-        *  Map from botUserId to accessTokens.
+        * Access token managment
         */
-        accessTokens: PropTypes.object.isRequired,
+        createUserAccessToken: (userId: string, description: string) => Promise<{
+            data: {token: string; description: string; id: string; is_active: boolean} | null;
+            error?: Error;
+        }>;
+
+        revokeUserAccessToken: (tokenId: string) => Promise<{data: string; error?: Error}>;
+        enableUserAccessToken: (tokenId: string) => Promise<{data: string; error?: Error}>;
+        disableUserAccessToken: (tokenId: string) => Promise<{data: string; error?: Error}>;
 
         /**
-        *  Map from botUserId to owner.
+        * Load owner of bot account
         */
-        owners: PropTypes.object.isRequired,
+        getUser: (userId: string) => void;
 
         /**
-        *  Map from botUserId to user.
+        * Disable a bot
         */
-        users: PropTypes.object.isRequired,
-
-        createBots: PropTypes.bool,
-
-        actions: PropTypes.shape({
-
-            /**
-            * Ensure we have bot accounts
-            */
-            loadBots: PropTypes.func.isRequired,
-
-            /**
-            * Load access tokens for bot accounts
-            */
-            getUserAccessTokensForUser: PropTypes.func.isRequired,
-
-            /**
-            * Access token managment
-            */
-            createUserAccessToken: PropTypes.func.isRequired,
-            revokeUserAccessToken: PropTypes.func.isRequired,
-            enableUserAccessToken: PropTypes.func.isRequired,
-            disableUserAccessToken: PropTypes.func.isRequired,
-
-            /**
-            * Load owner of bot account
-            */
-            getUser: PropTypes.func.isRequired,
-
-            /**
-            * Disable a bot
-            */
-            disableBot: PropTypes.func.isRequired,
-
-            /**
-            * Enable a bot
-            */
-            enableBot: PropTypes.func.isRequired,
-        }),
+        disableBot: (userId: string) => Promise<ActionResult>;
 
         /**
-        *  Only used for routing since backstage is team based.
+        * Enable a bot
         */
-        team: PropTypes.object.isRequired,
-    }
+        enableBot: (userId: string) => Promise<ActionResult>;
+    };
+
+    /**
+    *  Only used for routing since backstage is team based.
+    */
+    team: Team;
+}
+
+type State = {
+    loading: boolean;
+}
 
-    constructor(props) {
+export default class Bots extends React.PureComponent<Props, State> {
+    public constructor(props: Props) {
         super(props);
 
         this.state = {
             loading: true,
         };
     }
 
-    componentDidMount() {
+    public componentDidMount(): void {
         this.props.actions.loadBots(
             Constants.Integrations.START_PAGE_NUM,
-            Constants.Integrations.PAGE_SIZE,
+            Constants.Integrations.BOTS_PER_PAGE_DEFAULT,
```

Comment:
loadBots function expects a number as its second arg (PAGE_SIZE is a string of '1000'). I copied over the constant BOTS_PER_PAGE_DEFAULT from mattermost-redux to utils/constants.

Reply:
Are we able to just do a cast or a `parseInt` here instead? I don't think we should have 2 copies of the same constant in case we ever need to change it.

Reply:
The other option would be to modify the constant in `mattermost-redux` to be a `number` instead.